package.json

```diff
@@ -48,8 +48,10 @@
   "scripts": {
     "clean": "yarn lerna run --no-bail clean",
     "check-dependencies": "node ./scripts/check-mismatched-dependencies.cjs",
-    "docs": "typedoc --tsconfig tsconfig.build.json",
+    "docs": "run-s docs:build docs:update-functions-path",
+    "docs:build": "typedoc --tsconfig tsconfig.build.json",
     "docs:markdown-for-agoric-documentation-repo": "typedoc --plugin typedoc-plugin-markdown --tsconfig tsconfig.build.json",
+    "docs:update-functions-path": "node ./scripts/update-typedoc-functions-path.cjs",
```

Comment:
confirming `update-functions-path` isn't needed for `markdown-for-agoric-documentation-repo`

Reply:
I'm not certain here - are we currently using `docs:markdown-for-agoric-documentation-repo`? If yes, it seems this should be updated.

Reply:
See [c536276](https://github.com/Agoric/agoric-sdk/pull/9727/commits/c536276c4e4195b020bbbd954abd8d7850ffb54a)

Reply:
> See [c536276](https://github.com/Agoric/agoric-sdk/pull/9727/commits/c536276c4e4195b020bbbd954abd8d7850ffb54a)

That was naive - I forgot automated testing in this repo doesn't cover this path. Please see [5cf3e0f](https://github.com/Agoric/agoric-sdk/pull/9727/commits/5cf3e0ff77b55678535eedf8987da9f5498f8a0a) which adds required changes to process markdown files.

I tested manually. Unsure if there's a path for testing in agoric/documentation before merging.